data/common.yaml

```diff
@@ -80,3 +80,7 @@ profile::platform::baseline::enable_monitoring: false
 ##
 # Puppet Enterprise
 puppet_enterprise::profile::master::check_for_updates: false
+
+profile::app::pipelines::agent::access_token: 'ENTER_VALUE'
+profile::app::pipelines::agent::secret_key: 'ENTER_VALUE'
+profile::app::pipelines::agent::download_url: 'http://ENTER_VALUE'
```

Comment:
See comment on site/profile/manifests/app/pipelines/agent.pp. These should become required parameters, and not have (ineffective) default values.